<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Jyothi" failures="1" tests="1" name="cucumberoptions.TestNgTestRunner" time="5.426" errors="0" timestamp="2022-12-25T00:18:43 IST" skipped="0">
  <testcase classname="cucumberoptions.TestNgTestRunner" name="runScenario" time="5.426">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at stepDefination.GitPracticeStepDefination.user_moved_to_flight_booking_page(GitPracticeStepDefination.java:21)
at âœ½.User moved to Flight Booking page(file:///C:/Users/jinka/GitStuff/src/test/java/features/gitpractice.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- cucumberoptions.TestNgTestRunner -->
